_apps/auto-merge.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Auto Merge
+description: Automatically merges pull requests based on conditions
+slug: probot-auto-merge
+screenshots:
+- https://user-images.githubusercontent.com/6375609/45245043-d9c5ff00-b2fa-11e8-8745-2588323edda8.png
+authors: [ bobvanderlinden ]
+repository: bobvanderlinden/probot-auto-merge
+host: https://probot-auto-merge-production.herokuapp.com
+---
+
+Automatically merges pull requests based on configured conditions. It also updates and deletes branches when needed.
+
+## Usage
+
+1. [Configure the GitHub App](https://github.com/apps/probot-auto-merge)
+2. Create `.github/auto-merge.yml` in your repository.
+3. Customize configuration to your needs. See below.
+
+## Configuration
+
+Configuration of probot-auto-merge is done through `.github/auto-merge.yml` in
+your repository. An example of this file can be found [here](https://github.com/bobvanderlinden/probot-auto-merge/auto-merge.example.yml).
+
+The configuration has values that serve as conditions on whether or not a pull request
+should be automatically merged and also configuration about the merge itself. Values
+that serve as conditions are annotated as such below.
+
+All conditions must be met before a PR will be automatically merged. You can get more
+flexibility by defining multiple rules. Rules can have multiple conditions and if any
+of the conditions inside a rule are met, the PR is also merged. See [rules](#Rules).
```

Comment:
I think `[rules](#Rules)` needs to be `[rules](#`rules` (default: none))` to work